### PR TITLE
Connect to api

### PR DESCRIPTION
**Pull request by Samuellekpor**
In this milestone, we:

1. Make API call
2. Add a function to get all the books
3. Add functions for _adding_ and _deleting_ a book

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,8 +29,6 @@ After you're finished please remove all the comments and instructions!
 - [🙏 Acknowledgements](#acknowledgements)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
-
 # 📖 [your_project_name] <a name="about-project"></a>
```
Comment (emmiiorji) [inline README.md]: [Optional] You have done great implementing the new readme template. I suggest that you replace `[your_project_name]` with your actual project name. The text is only a placeholder and guide provided by the template.

File: src/components/AddBook.js [hunk 2]
```diff
@@ -13,12 +13,13 @@ function AddBook() {
...
     const newBook = {
-      id: uuidv4(),
+      item_id: uuidv4(),
       title,
       author,
+      category: 'Action',
     };
 
-    dispatch(addBook(newBook));
+    dispatch(saveBook(newBook));
```
Comment (emmiiorji) [inline src/components/AddBook.js:22]: You have done great implementing API calls for the actions. Kindly ensure that the actions are dispatched as they should. Inspecting the network section of the chrome dev, there seems not to be an actual API call on adding a book. The same goes for removing a book. Kindly fix this.

image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Samuellekpor, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

A little bit more


## Good Highlights✔️

- Linter checks are passing
- Descriptive PR
- Readme is professional


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddBook.js (hunk 2 above)
Comment (Samuellekpor) [reply src/components/AddBook.js:22]: Hello @emmiiorji!! 👋🏾 
Thank you for the quick review. I actually made API requests on the delete and add buttons. I just repeated the actions of adding a book and removing one. And I can see the results on POSTMAN with the book added when I add and another one deleted whenever I click on the REMOVE button. And there is no way in the code to add the book to the page if it doesn't come from the API. Please check again

File: README.md (hunk 1 above)
Comment (Samuellekpor) [reply README.md]: Thanks for the comment 🙏🏾 . I will quickly fix it 💯

**Review by FrederickMih — APPROVED** [review]
Hi @Samuellekpor, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- API calls has been made :+1: 
- No Linter error :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**